Formula/angband.rb

```diff
@@ -6,6 +6,11 @@ class Angband < Formula
   license "GPL-2.0"
   head "https://github.com/angband/angband.git"
 
+  livecheck do
+    url :head
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
```

Comment:
```suggestion
    url :homepage
    regex(/href=.*?angband[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

The `homepage` links to the `stable` archive and we should use that so that the check is aligned with the `stable` source.

We only create a `livecheck` block that uses `head` as a last resort when other sources don't work properly. We only have a handful of `livecheck` blocks that use `url :head` (around 30) and I've been actively reducing that number recently.